### PR TITLE
Fixed field declarations which should be readonly

### PR DESCRIPTION
**Pull request by jp2masa**
## Changes
- Fixed field declarations which should be readonly.

This is a breaking change, but I don't believe there is anyone setting these fields to a different value.

### CODE REVIEW

**Comment by wieslawsoltes** [comment]
I set Color property on gradient stop and I would guess other can be used
also.

śr., 21 lis 2018, 02:32: jp2masa <notifications@github.com> napisał(a):

**Comment by jp2masa** [comment]
@wieslawsoltes `Color` or `ColorProperty`?

**Review by grokys — APPROVED** [review]

**Review by kekekeks — DISMISSED** [review]

File: src/Avalonia.Input/DataFormats.cs [hunk 1]
```diff
@@ -5,11 +5,11 @@ public static class DataFormats
         /// <summary>
         /// Dataformat for plaintext
         /// </summary>
-        public static string Text = nameof(Text);
```
[the remark this answers is not in the document]
Comment (jp2masa) [reply src/Avalonia.Input/DataFormats.cs]: Should I revert these changes?

[the remark this answers is not in the document]
Comment (kekekeks) [reply src/Avalonia.Input/DataFormats.cs]: If some already library compiled library references there fields the usage of said library with your changes will result in TypeLoadException.

**Review by grokys — APPROVED** [review]

**Pull request merged**